src/id.rs

```diff
@@ -21,9 +21,12 @@ pub struct Id(NonZeroU64);
 impl Id {
     /// Allocate a new, unique `Id`.
     pub fn next() -> Id {
```

Comment:
`Id` is kind of an awful name for this type, although this PR isn't really the place to fix it

Reply:
Yeah, I think the mechanism is maybe not ideal, and personally think the type should maybe be `Option<NonZeroUsize>` (but that's technically complicated by AccessKit, which wants `u64`), and this patch could introduce a needless runtime check when coercing to NonZero (which is just `unsafe` in Glazier) depending on how aware the compiler is of the static initalizer, but it's fine for now I think.

Reply:
I've turned this into an issue so as to avoid blocking this PR https://github.com/linebender/xilem/issues/168